### ISSUE TITLE
Black screen after install, no boot past ram disk init

### ISSUE THREAD

**Issue by Kirkland-Pickles** [zweiler2/HoloISO_Nvidia#26]
## Your Issue: 
```no boot
*type your problem here*
```after initializing ram disk the screen just stays blank

## Your device's full specs
```
CPU: i9 11900kf
GPU: *rtx 3080*
```

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
snapshotxxxxxxxxxxx_xxxx
```

## (Only for SteamOS Session loading issues) Have you tried launching `gamescope-session` from terminal? If yes, provide a log.
```
*Please put anything here if you tried to launch session separately*
```

## Have you updated your installation recently?
```
*yes or no*
```

**Comment by Naymmmm** [comment]
Same issue.

**Comment by Kirkland-Pickles** [comment]
> Same issue.

Yeah, it seems getting any kind of steamos to work on nvidia right now is impossible. I tried older versions of this tweak, and while they did work, they have impossibly slow and laggy UI in game mode. 
I tried about 10 other distros and all give similar or worse results.
I went back to Windows with bpm on startup.
Considering swapping my 3080 for a cheap AMD GPU for steamos alone, idk.

**Comment by Naymmmm** [comment]
> > Same issue.
> 
> Yeah, it seems getting any kind of steamos to work on nvidia right now is impossible. I tried older versions of this tweak, and while they did work, they have impossibly slow and laggy UI in game mode. I tried about 10 other distros and all give similar or worse results. I went back to Windows with bpm on startup. Considering swapping my 3080 for a cheap AMD GPU for steamos alone, idk.

I'm running it on VMWare, probably a me issue.

**Comment by zweiler2** [comment]
@Benjamin8obadiah Probably a Nvidia GPU problem. Does this issue persist in the newest ISO?

@Naymmmm VMs are not supported, because gamescope needs a Vulkan capable GPU.
I'd suggest you try to pass through a GPU into the VM.

**Comment by ValkVFX** [comment]
any progress to make it work?
i get past it sometimes but it gives me an out of memory error
